feat(i18n): add Czech translations

jakubhruby opened the pull request:
## **User description**
### SUMMARY
Added czech language translations

### TESTING INSTRUCTIONS
Turn on CS lang and check applied phrases

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [x] Introduces new feature or API
- [ ] Removes existing feature or API

originally requested in https://github.com/apache/superset/pull/35639 (changed source branch)


___

## **CodeAnt-AI Description**
**Add Czech language support and translations**

### What Changed
- Added Czech as a selectable UI language
- Added Czech translations for supported interface text

### Impact
`✅ Czech-language UI`
`✅ Fewer untranslated screens`
`✅ Clearer language selection`
<details>
<summary><strong>💡 Usage Guide</strong></summary>

### Checking Your Pull Request
Every time you make a pull request, our system automatically looks through it. We check for security issues, mistakes in how you're setting up your infrastructure, and common code problems. We do this to make sure your changes are solid and won't cause any trouble later.

### Talking to CodeAnt AI
Got a question or need a hand with something in your pull request? You can easily get in touch with CodeAnt AI right here. Just type the following in a comment on your pull request, and replace "Your question here" with whatever you want to ask:
<pre>
<code>@codeant-ai ask: Your question here</code>
</pre>
This lets you have a chat with CodeAnt AI about your pull request, making it easier to understand and improve your code.

#### Example
<pre>
<code>@codeant-ai ask: Can you suggest a safer alternative to storing this secret?</code>
</pre>

### Preserve Org Learnings with CodeAnt
You can record team preferences so CodeAnt AI applies them in future reviews. Reply directly to the specific CodeAnt AI suggestion (in the same thread) and replace "Your feedback here" with your input:
<pre>
<code>@codeant-ai: Your feedback here</code>
</pre>
This helps CodeAnt AI learn and adapt to your team's coding style and standards.

#### Example
<pre>
<code>@codeant-ai: Do not flag unused imports.</code>
</pre>

### Retrigger review
Ask CodeAnt AI to review the PR again, by typing:
<pre>
<code>@codeant-ai: review</code>
</pre>

### Check Your Repository Health
To analyze the health of your code repository, visit our dashboard at [https://app.codeant.ai](https://app.codeant.ai). This tool helps you identify potential issues and areas for improvement in your codebase, ensuring your repository maintains high standards of code health.

</details>

jakubhrubyorgis committed:
cs translations

jakubhrubyorgis committed:
fix wording

jakubhruby committed:
Merge branch 'apache:master' into master

jakubhruby committed:
Merge branch 'apache:master' into master

Commit:
Merge branch 'githubMaster' into feat/csTranslations

Commit:
Merge branch 'githubMaster' into feat/csTranslations

jakubhruby committed:
Merge branch 'apache:master' into feat/csTranslations

Commit:
remove merge dupicity

davidkopelent commented:
Hello @jakubhruby I addressed the comments from the bito bot. Are you okay if I push them? Also I don't see any failing test anymore, just that the workflows need approval to run, does it mean that we are okay to request a review from some maintainer?

jakubhruby commented:
> Hello @jakubhruby I addressed the comments from the bito bot. Are you okay if I push them? Also I don't see any failing test anymore, just that the workflows need approval to run, does it mean that we are okay to request a review from some maintainer?

Yes, no problem

davidkopelent committed:
chore: address bito comments on translations

davidkopelent commented:
@jakubhruby I tried to push, but don't have the permission to do so. Would it be possible to grant me the permission please? Or if not I can open a PR on your fork if that is more suitable for you.

jakubhruby commented:
@davidkopelent I invited you as a collaborator

davidkopelent committed:
Merge branch 'master' into feat/csTranslations

davidkopelent commented:
Hello @rusackas can you please review this PR or point me to someone who can? Thank you

sfirke commented on superset/translations/cs/LC_MESSAGES/messages.po:
```diff
@@ -0,0 +1,17231 @@
+msgid "A list of tags that have been applied to this chart."
+msgstr "Seznam značek, které byly aplikovány na tento graf."
+
+#, fuzzy
+msgid "A list of tags that have been applied to this dashboard."
+msgstr "Seznam štítků, které byly aplikovány na tento graf."
```
Bito is speaking Czech? 🤯

jakubhruby committed:
Merge branch 'master' into feat/csTranslations

sfirke commented:
@rusackas this one is also green and ready to go

jakubhruby committed:
Merge branch 'master' into feat/csTranslations

jakubhruby committed:
Merge branch 'master' into feat/csTranslations

rusackas reviewed (approved):
Thanks for the updates and patience here.

rusackas reviewed (approved):
Thanks @jakubhruby and @davidkopelent — adding a new language is always welcome. CI green, already approved by @Vaek, and the non-translation changes are minimal (one entry in `LANGUAGES` and a labeler rule). The bito bot flagged some translation-accuracy nits and fuzzy entries, but those are exactly the kind of thing Weblate is for and shouldn't block landing a new locale. LGTM.
